tests/unit/test_mysql.py

```diff
@@ -103,3 +108,44 @@ def test_1971565(self, mock_helper):
             ]
         }
         self.assertEqual(IssuesManager().load_bugs(), expected)
+
```

Comment:
looks like you accidentally delete test test_lp372017 that is in your prior patchset.